### DISCUSSION TITLE
When dpgen ran 02.fp,  the following error: AttributeError: 'list' object has no attribute 'shape'

### DISCUSSION THREAD

**Discussion by hanrm94** [deepmodeling/dpgen#1412] [Q&A]
The following screenshot shows the error message.
If you can, please provide me with a detailed solution.
Thank you for your help.
<img width="492" alt="屏幕截图 2023-11-24 165812" src="https://github.com/deepmodeling/dpgen/assets/133116023/5c0583cb-6a5b-4da0-9018-c71b628921e3">

**Comment by njzjz** [comment]
See #1237
